### ISSUE TITLE
NameError: name 'e' is not defined

### ISSUE THREAD

**Issue by cgreene** [AlexsLemonade/refinebio#569]
Looks like this line might need to use `as e`

https://sentry.io/greenelab/staging-refinebio/issues/667311972/

```
AttributeError: 'NoneType' object has no attribute 'open'
  File "data_refinery_workers/processors/salmon.py", line 195, in _find_or_download_index
    with index_tarball.open(file.get_synced_file_path(), "r:gz") as index_archive:

NameError: name 'e' is not defined
  File "data_refinery_workers/processors/utils.py", line 186, in run_pipeline
    last_result = processor(last_result)
  File "data_refinery_workers/processors/salmon.py", line 200, in _find_or_download_index
    except e:

Unhandled exception caught while running processor function _find_or_download_index in pipeline
```

**Comment by cgreene** [comment]
The instance of this in `salmon.py` gets closed by #540. Getting the one in `utils.py` will complete closing this.

**Comment by kurtwheeler** [comment]
I don't think there's an issue in `utils.py`. I believe that is just mentioned because it is higher up in the stack trace.

**Comment by cgreene** [comment]
even better :)

**Comment by jaclyn-taroni** [comment]
I believe this was closed by #540. Since we're planning to clear out Sentry before our next test crunch, I assume it will pop up again if there's an issue in `utils.py`.
